packages/typespec-ts/src/utils/modelUtils.ts

```diff
@@ -70,33 +70,39 @@ import {
   getDefaultApiVersion,
   isApiVersion
 } from "@azure-tools/typespec-client-generator-core";
-import { SdkContext } from "./interfaces.js";
+import { GetSchemaOptions, SdkContext } from "./interfaces.js";
 import { getModelNamespaceName } from "./namespaceUtils.js";
+import { KnownMediaType, hasMediaType } from "./mediaTypes.js";
+
+export const BINARY_TYPE_UNION =
```

Comment:
What is our coding style to define those unions:  BINARY_TYPE_UNION, BINARY_AND_FILE_TYPE_UNION ?

Reply:
emm... I don't think we have an agreement for the detailed code style but here i use the google code style for constant.
https://google.github.io/styleguide/jsguide.html#naming-constant-names